gradle.properties

```diff
@@ -1,8 +1,8 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=2.3.0-SNAPSHOT
+version=3.0.0-SNAPSHOT
```

Comment:
Do we need to change version?

Reply:
Probably a major version needed to be released with Java 17 support. For now I'll restore the changes.